lib/storekit/receipt.rb

```diff
@@ -1,19 +1,42 @@
+require 'ostruct'
+
 module StoreKit
-  class Receipt
-    attr_reader :receipt_data
+  class Receipt < OpenStruct
+    TRANSFORM = {
+       'unique_identifier' => true,
+       'product_id' => true,
+       'bid' => true,
+       'unique_vendor_identifier' => true,
+       'bvrs' => true,
+       'transaction_id' => proc { |v| v.to_i },
+       'original_transaction_id' => proc { |v| v.to_i },
+       'quantity' => proc { |v| v.to_i },
+       'item_id' => proc { |v| v.to_i },
+       'web_order_line_item_id' => proc { |v| v.to_i },
```

Comment:
Let's see into the example. In documentation we've seen:

> This receipt is valid but the subscription has expired. When this status code is returned to your server, the receipt data is also decoded and returned as part of the response.

That is the full error text with no additional markings. So why you didn't add the full text into the as you've advocated for?